src/os/bluestore/BlueStore.cc

```diff
@@ -4474,6 +4475,10 @@ int BlueStore::fsck(bool deep)
 	  continue; // go for next object
 	}
 	used_nids.insert(o->onode.nid);
+      } else {
+        derr << __func__ << " " << oid << " has uninitialized nid"
+             << dendl;
+        ++errors;
```

Comment:
We only set the nid in certain paths... and we only use it for omap.  If we do this, we should consolidate all the assign_nid calls to the points where we set exists = true.